activities/MazeWeb.activity/js/activity.js

```diff
@@ -1,6 +1,19 @@
 define(["sugar-web/activity/activity","tween","rAF","activity/directions","sugar-web/graphics/presencepalette", "sugar-web/env",  "sugar-web/graphics/icon", "webL10n", "sugar-web/graphics/palette", "rot", "humane"], function (activity, TWEEN, rAF, directions, presencepalette, env, icon, webL10n, palette, ROT, humane) {
 
-    requirejs(['domReady!'], function (doc) {
+    // Resize screen
+  document.getElementById("fullscreen-button").addEventListener('click', function() {
+    console.log('RESIZE')    
+    document.getElementById("main-toolbar").style.opacity = 0;
+    document.getElementById("canvas").style.top = "0px";
+    document.getElementById("unfullscreen-button").style.visibility = "visible";
```

Comment:
wouldnt be better to use `display: none` instead of `visibility: hidden`?

Reply:
for some reason, the display:none didn't work...

Reply:
done